TC/iocBoot/iocTC-IOC-01/st-common.cmd

```diff
@@ -19,6 +19,10 @@ epicsEnvSet("ADS_PORT", $(ADS_PORT=852))
 # Get the number of axes from the controller before we starting spinning up dbs for each axis. the NUM_AXES macro is set to this number by getAdsVar() 
 getAdsVar("NUM_AXES", "GVL_APP.nAXIS_NUM", "$(IP_AD)", "$(AMS_ID)", $(ADS_PORT))
 
+# If the above didn't work, exit now to avoid trying to autosave incorrect values
+stringiftest(“CONNECTED”, “$(NUM_AXES=)”)
```

Comment:
was this pasted from an email? I think the quotes marks are the wrong ascii character - they look different to line 20 etc.

Reply:
it was - will blame outlook for using weird quotes!